YoutubeExplode/Playlists/Playlist.cs

```diff
@@ -13,7 +13,7 @@ public class Playlist : IPlaylist
     public PlaylistId Id { get; }
 
     /// <inheritdoc />
-    public string Url => $"https://www.youtube.com/playlist?list={Id}";
+    public string Url => _url ?? $"https://www.youtube.com/playlist?list={Id}";
```

Comment:
Would `https://www.youtube.com/watch?list=RDTsYhxMnGYCw` work? To avoid having to pass first video ID.

Reply:
It does, but the first video shows as unavailable on the current device. So i think is better to pass the first video ID in the url in this case.